src/Sarif/SdkResources.resx

```diff
@@ -226,10 +226,10 @@
     <value>Unexpected fatal runtime condition(s) observed: </value>
   </data>
   <data name="MSG_OneOrMoreInvalidTargets" xml:space="preserve">
-    <value>One or more targets was skipped entirely as it was determined to be an invalid target for analysis. Pass --verbose on the command-line for more information.</value>
+    <value>One or more targets was skipped entirely as it was determined to be an invalid target for analysis. Include all supported values for both --kind and --level on the command-line for more information.</value>
```

Comment:
>Include all supported values for both --kind and --level [](http://example.com/codeflow?start=111&length=56)

I believe that `--kind Fail;NotApplicable` is the specific value required. You don't need `--level` here as this severity level isn't relevant except for kind == Fail. Can you test this?

Here's a new sentence for you if I'm correct in what I assume:

Include `NotApplicable` on an explicit `--kind` command-line argument (e.g., `--kind Fail;NotApplicable`) for more information.

Reply:
tested and updated. 
MSG_OneOrMoreInvalidTargets and 
MSG_OneOrMoreNotApplicable is different,
I have tested the specific value required.

Reply:
oh btw I did not use the markup backtick, we don't use markup in the console output currently and i think that is correct.